en/messages.json

```diff
@@ -298,6 +298,37 @@
         "message": "With Relay Premium, you can block promotional emails from reaching your inbox while still allowing you to receive emails like receipts or shipping information.",
         "description": "DO NOT TRANSLATE \"Relay Premium\"."
     },
+    "popupBundlePromoHeadline": {
+        "message": "Add Mozilla VPN at $SAVINGS$ off.",
+        "description": "DO NOT TRANSLATE \"Mozilla VPN\"",
+        "placeholders": {
+            "savings": {
+                "content": "50%"
+            }
+        }
+    },
```

Comment:
Is this the right way to call out variables?

Reply:
In checking how other variables are referenced in the file, it looks good!